### PR TITLE
feat(FEC-14): added different libs and helpers

### PR DESCRIPTION
**Pull request by cTux**

### CODE REVIEW

File: src/utils/axios.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+import { baseURL } from '~/utils/axios.config';
+
+export const axiosInstance = axios.create({
+  baseURL,
+});
```
Comment (MaxKalashnyk) [inline src/utils/axios.ts:7]: How about adding Axios interceptors here? [Here](https://stackoverflow.com/questions/52737078/how-can-you-use-axios-interceptors) are a couple of examples.

File: src/utils/localStorage.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+export const getFromLocalStorage = <T>(key: string): T | null => {
```
Comment (MaxKalashnyk) [inline src/utils/localStorage.ts:1]: We can also add a helper hook for working with the localStorage. What do you think?
Here is code of this hook:
``` typescript
import { useState, useEffect } from "react";

export const useStorageState = <S = undefined>(key: string, initial: S) => {
  const [value, setValue] = useState<S>(() => {
    const saved = localStorage.getItem(key);
    if (saved) {
      return JSON.parse(saved);
    }
    return initial;
  });

  useEffect(() => {
    localStorage.setItem(key, JSON.stringify(value));
  }, [value]);

  return [value, setValue] as [typeof value, typeof setValue];
};
```

**Review by MaxKalashnyk — APPROVED** [review]
LGTM! I've left a couple of minor comments

File: src/utils/localStorage.ts (hunk 2 above)
Comment (cTux) [reply src/utils/localStorage.ts:1]: Sure!

File: src/utils/axios.ts (hunk 1 above)
Comment (cTux) [reply src/utils/axios.ts:7]: Let's move it to cookbook if you don't mind. And put a few examples how to add interceptors for different authorizations.

**Pull request merged**